auth-api/src/auth_api/services/entity.py

```diff
@@ -102,6 +103,8 @@ def save_entity(entity_info: dict):
         if existing_entity is None:
             entity_model = EntityModel.create_from_dict(entity_info)
         else:
+            # TODO temporary allow update passcode, should replace with reset passcode endpoint.
```

Comment:
Created this ticket to address this: https://github.com/bcgov/entity/issues/1480